Examples/CaseStudies/SwiftUICaseStudies/01-GettingStarted-Animations.swift

```diff
@@ -47,7 +47,7 @@ struct AnimationsState: Equatable {
 
 enum AnimationsAction: Equatable {
   case circleScaleToggleChanged(Bool)
-  case alertDismissed
```

Comment:
I think that what @stephencelis what suggesting was to reorder cases in the alphabetical order:
```swift
case alertDismissed
case circleScaleToggleChanged
…
```

Reply:
That's what he meant. Thanks @tgrapperon ! 😄 just edited last commit.